polygeist/tools/cgeist/Lib/clang-mlir.cc

```diff
@@ -925,235 +914,6 @@ ValueCategory MLIRScanner::CommonArrayLookup(ValueCategory array,
   return ValueCategory(post, /*isReference*/ true);
 }
 
-std::pair<ValueCategory, bool>
```

Comment:
Moved to CGCall.cc as it is called from `VisitCallExpr` which is already in that file.